release-notes/major10/10.8.0/valtimo-frontend-libraries.md

```diff
@@ -6,11 +6,13 @@ The following features were added:
 
 * **IDs added to menu items and search field buttons**
 
-  IDs can be configured for menu items in the ValtimoConfig by adding the optional 'id' property. IDs have been added to buttons in the searchfields component.
+  - The menu item configuration in the Angular environment file now supports an optional 'id' property. When configured, the 'id' property is added to the HTML element.
+  - 'id' attributes have been added to the 'clear' and 'search' buttons on the case list page.
 
-* **Added due date configuration to portaal taak plugin**
+* **Added due date configuration to portaaltaak plugin**
```

Comment:
Earlier it's portaal taak, here it's portaaltaak. Which of the two is it? I think we should be consistent